x-pack/plugins/security_solution/public/common/components/event_details/cti_details/enrichment_summary.tsx

```diff
@@ -134,7 +134,10 @@ const EnrichmentSummaryComponent: React.FC<{
 
     const fieldsData: FieldsData = {
       field: field ?? '',
-      format: browserField?.format ?? '',
+      format:
+        typeof browserField?.format === 'string'
```

Comment:
would it be a good idea to use a `getBrowserFieldFormat` or something, and then you can do a find and replace on the util to `browserField?.format?.id` once the spec is completely aligned?

Reply:
I am curious as to why we cannot use original `browserField.format` without this transformation and do this transformation either at the original or at the bottom of the tree where this format is used?

Reply:
done here and everywhere else I had to do this condition, you are right:)